Test/FloatTorchTensorUnitTest.cs

```diff
@@ -0,0 +1,99 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System;
+using TorchSharp;
+
+namespace Test
+{
+    [TestClass]
+    public class FloatTorchTensorUnitTest
+    {
+        [TestMethod]
+        public void TestCreation0D()
+        {
+            var x = FloatTorchTensor.Create();
```

Comment:
Here, and elsewhere, maybe use `Tensor<Float>` instead of `var` to make it obvious you are accessing it via that interface.

Reply:
fixed